resources/popup.js

```diff
@@ -2,11 +2,14 @@
 function toggle() {
     chrome.storage.sync.get({
         loggingEnabled: true
-    }, function (res) {
-        chrome.storage.sync.set({
-            loggingEnabled: !(res.loggingEnabled)
-        })
-    });
+    }, function (res) { toggleSetter(res) });
```

Comment:
Can't this be written like `chrome.storage.sync.get({loggingEnabled: true}, toggleSetter)`? (maybe with some whitespaces/newlines)